README.md

```diff
@@ -77,11 +77,11 @@ const totalPrice = 500; // Pass in the total price in smallest currency unit (ie
 const currency = 'USD';
 patch.orders.createOrder({ total_price: totalPrice, currency: currency });
 
-// Create order with the issuedTo field (optional)
+// Create order with the issued_to field (optional)
 const amount = 1_000_000; // Pass in the amount in unit specified
 const unit = 'g';
-const issuedTo = { email: 'issuee@companya.com', name: 'Olivia Jones' };
-patch.orders.createOrder({ amount: amount, unit: unit, issuedTo: issuedTo });
+const issued_to = { email: 'issuee@companya.com', name: 'Olivia Jones' };
+patch.orders.createOrder({ amount: amount, unit: unit, issued_to: issued_to });
```

Comment:
Hum, for `node.js`, I'm 99% sure that the SDK actually requires `issuedTo`, not `issued_to`.

Reply:
Yup: https://github.com/patch-technology/patch-node/blob/main/src/model/CreateOrderRequest.js#L74

Not a fan of the generator changing the case from the API parameter, but not sure it's worth fighting the generator for.